### ISSUE TITLE
Condition typo in skin xml

### ISSUE THREAD

**Issue by jbaanus** [osmc/osmc#195]
In osmc/package/mediacenter-skin-osmc/files/usr/share/kodi/addons/skin.osmc/16x9/Variables.xml there is a typo on line 91:

```
		<value condition="Window.IsVisible(filemanager)]">$LOCALIZE[7]</value>
```

Spot the "]" character at the end of the condition. I am getting errors in the log:

```
01:49:53 T:1956671488   ERROR: Unmatched ]
01:49:53 T:1956671488   ERROR: Error parsing boolean expression Window.IsVisible(filemanager)]
01:49:53 T:1956671488   ERROR: Window Translator: Can't find window settingsprofile
```

**Comment by samnazarko** [comment]
Thanks for reporting this.

**Comment by amchoukir** [comment]
I still see the issue did your fix get released? I fixed by hand for now.

```
osmc@osmc:/$ dpkg --get-selections | grep mediacenter-skin-osmc
mediacenter-skin-osmc				install
osmc@osmc:/$ dpkg -s mediacenter-skin-osmc
Package: mediacenter-skin-osmc
Status: install ok installed
Priority: optional
Section: video
Installed-Size: 26118
Origin: OSMC
Maintainer: Sam G Nazarko <email@samnazarko.co.uk>
Architecture: all
Version: 3.1.0-3
Description: OSMC skin for mediacenter
```

**Comment by samnazarko** [comment]
This was fixed a few months ago

**Comment by amchoukir** [comment]
Yes I know from your diff those fixes should have been picked up as part of package version `3.0.5-2` as per your diff but I am on `3.1.0-3` and I see the issue. So wanted to make sure that either the chnages were indeed picked by some package version before `3.1.0-3` or whether the file are correctly overwritten on package upgrade.

**Comment by fzinken** [comment]
The error of the wrongly place ']' in the skin is fixed as you can see below
```
osmc@osmc.my.hk:~$ dpkg -l | grep skin
ii  mediacenter-skin-osmc                3.1.0-4                           all          OSMC skin for mediacenter
osmc@osmc.my.hk:~$ cat /usr/share/kodi/addons/skin.osmc/16x9/Variables.xml | grep "Window.IsVisible(filemanager)"
		<value condition="Window.IsVisible(filemanager)">$LOCALIZE[7]</value>
```

I suggest 1st update to the latest skin and if you still see the error in the code let us know

**Comment by amchoukir** [comment]
Ok so what I did is that I reintroduced the error:

<value condition="Window.IsVisible(filemanager)]">$LOCALIZE[7]</value>

I moved my source.list to jessie-devel to get `3.1.0-4` and upgraded only that package. It indeed fixed the issue. I don't know why this did not happen for my raspberry in the upgrades from `3.0.5-2` to `3.1.0-3`.

Thanks @fzinken

**Comment by amchoukir** [comment]
Actually now I came back to Jessie and downgraded the package to `3.1.0-3` and the error is there:

```
<value condition="Window.IsVisible(filemanager)]">$LOCALIZE[7]</value>
```

This means that the fix was only actually shipped in `3.1.0-4`, it is a bit weird given that the fix is a few months old, no?
